newsrevenuehub/rev-engine
# [NRH-267.270] Elements: Update donor info + Donor address + Amount >= 1

- [x] Merge NRH-263 in to develop
- [x] Fix tests for this PR

#### What's this PR do?
Implements NRH-267, NRH-270, NRH-271.
1. The "Donor info" block is now just name and email. It is required, can only have one, and all fields must be filled out to donate.
2. New "Donor address" block contains address info, including zip and country. It is required, can only have one, and all fields must be filled out to donate.
3. Donors will see a message "Please enter an amount of at least $1" if they have entered an "Other" amount less than 1.

#### How should this be manually tested?
Edit a donation page and add Donor info and Donor address blocks.
View that page live and try to submit without filling anything out.
Note that error messages appear beneath the required fields.
Also, try to donate $0.50 using the "Other" field. Observe that a message appears instead of the card payment form.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No
